serverless-functions/ReadSourceData/__init__.py

```diff
@@ -0,0 +1,65 @@
+import json
+import azure.functions as func
+import logging
+
+from phdi.harmonization.hl7 import (
+    convert_hl7_batch_messages_to_list,
+)
+
+
+def main(blob: func.InputStream, queue: func.Out[str]) -> None:
+    """
+    When this function is triggered with a blob payload, read the new file if its
+    name begins with 'source-data/', identify each individual messsage
+    (ELR, VXU, or eCR) contained in the file, and publish each one to a GCP pubsub
```

Comment:
May want to change this from GCP to Azure.  Also, does Azure have a pubsub topic?